nucypher/cli/painting/help.py

```diff
@@ -74,6 +75,38 @@ def paint_new_installation_help(emitter, new_configuration):
     character_name_starts_with_vowel = character_name[0].lower() in vowels
     adjective = 'an' if character_name_starts_with_vowel else 'a'
     suggested_command = f'nucypher {character_name} run'
-    how_to_run_message = f"\nTo run {adjective} {character_name.capitalize()} node from the default configuration filepath run: \n\n'{suggested_command}'\n"
+    how_to_run_message = f"\nTo run {adjective} {character_name.capitalize()} node from the default configuration " \
+                         f"filepath run: \n\n'{suggested_command}'\n"
 
     emitter.echo(how_to_run_message.format(suggested_command), color='green')
+
+
+def paint_probationary_period_disclaimer(emitter):
```

Comment:
The fact that we have this method called "paint_probationary_period_disclaimer" is kinda crazy.